activerecord/lib/active_record/encryption/encryptable_record.rb

```diff
@@ -30,6 +30,8 @@ module EncryptableRecord
         #   will use the oldest encryption scheme to encrypt new data by default. You can change this by setting
         #   <tt>deterministic: { fixed: false }</tt>. That will make it use the newest encryption scheme for encrypting new
         #   data.
+        # * <tt>:support_unencrypted_data</tt> - If `config.support_unencrypted_data` is +true+, you can set this to +false+
+        #   to opt out of unencrypted data support for this specific attribute.
```

Comment:
Maybe you can add a sentence about when this is useful: scenarios where you encrypt one table, and want to disable support for unencrypted data without having to tweak the global setting.